src/utils/havoc.js

```diff
@@ -564,3 +588,9 @@ export class HavocImplementation {
     }
   }
 }
+
+export class MaterializeImplementation {
+  materialize(realm: Realm, val: ObjectValue) {
+    materializeLeakedObject(realm, val);
```

Comment:
It's calling a function `...Leaked...` here. Is anyone checking the invariant that the object is marked as leaked?